CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Video
+- Xamarin Mac was upgraded so that 64-bit executables on macOS will be supported.
+- The `AudioPlayer` property of `VideoHandle` in the Mono CIL backend has been removed.
```

Comment:
`VideoHandle` is internal from Unos point of view, so this change shouldn't be user-visible.